### PR TITLE
San 5506 ping stream

### PR DESCRIPTION
**Pull request by anandkumarpatel**

### CODE REVIEW

File: lib/event-manager.js [hunk 1]
```diff
@@ -69,6 +69,14 @@ class EventManager {
   }
 
   /**
+   * @param  {String} host format: 10.0.0.1:4242
+   * @return {Listeners{}}
+   */
+  getListeners (host) {
+    return this.dockListeners
```
Comment (podviaznikov) [inline lib/event-manager.js:75]: why does this function has `host` if you don't use it?

File: lib/worker-server.js [hunk 2]
```diff
@@ -21,11 +21,13 @@ module.exports = new ponos.Server({
...
     'docker.event.publish': require('./workers/docker.event.publish.js'),
     'docker.events-stream.connect': require('./workers/docker.events-stream.connect.js'),
+    'docker.events-stream.ping': require('./workers/docker.events-stream.ping.js'),
     'swarm.events-stream.connect': require('./workers/swarm.events-stream.connect.js')
   },
   events: {
+    'instance.container.health-check.failed': require('./workers/instance.container.health-check.failed.js'),
     'swarm.events-stream.connected': require('./workers/swarm.events-stream.connected.js'),
-    'instance.container.health-check.failed': require('./workers/instance.container.health-check.failed.js')
+    'time.five-minutes.passed': require('./workers/time.five-minutes.passed.js')
```
Comment (podviaznikov) [inline lib/worker-server.js:30]: nice:)

File: lib/rabbitmq.js [hunk 3]
```diff
@@ -72,12 +75,21 @@ class Publisher extends RabbitMQ {
     })
   }
 
+  createPingJob (job) {
+    const log = logger.child({
+      job: job,
```
Comment (podviaznikov) [inline lib/rabbitmq.js]: just `job`

File: lib/rabbitmq.js [hunk 4]
```diff
@@ -72,12 +75,21 @@ class Publisher extends RabbitMQ {
...
+      method: 'createPingJob'
+    })
+    log.info('called')
+    this.publishTask('docker.events-stream.ping', job)
+  }
+
   createPublishJob (job) {
     const log = logger.child({
       job: job,
```
Comment (podviaznikov) [inline lib/rabbitmq.js]: just `job`

**Review by podviaznikov — COMMENTED** [review]
few minors. Would be good if you explain this PR in person

File: lib/event-manager.js (hunk 1 above)
Comment (anandkumarpatel) [reply lib/event-manager.js:75]: removing

File: lib/rabbitmq.js (hunk 3 above)
Comment (anandkumarpatel) [reply lib/rabbitmq.js]: +1

File: lib/rabbitmq.js (hunk 4 above)
Comment (anandkumarpatel) [reply lib/rabbitmq.js]: +1

**Review by podviaznikov — APPROVED** [review]

**Review by podviaznikov — APPROVED** [review]

**Pull request merged**